process/incident_management.md

```diff
@@ -0,0 +1,273 @@
+
+The post mortem meeting should involve all internal parties who were involved in the recovery itself and should be scheduled publicly to allow anyone interested to join and listen in or contribute:
+* [mandatory] The incident commander
+* [mandatory] The incident technical lead
+* [optional] Every SRE team members involved (as much as possible given the timezone constraints)
+  * The more brain power we have in the meeting the more likely we are to find the best countermeasures to avoid reoccurrence
+* [optional] The CEE lead
+
+The incident technical lead is the owner of the post mortem review meeting and is making sure that it contains the following:
+* Executive summary with customer impact, severity, …
+* Root Cause Analysis
+* Timeline of events
+* List of action items and countermeasures with
+  * Owner
+  * Criticality
+  * Target date for delivery
+
+All action items and countermeasures should aim at (as much as possible):
+
+* Avoid recurrence
+* Fix root cause
+* Improve monitoring and alerting by following best practices
+  * do not alert on edge cases, alert should be based on symptoms not root causes
+* Increase resiliency (being mindful of cost)
+* Increase automated recovery (if it makes sense)
+* Decrease MTTR in case of recurrence (documentation for instance, …)
+* **Each action item must be tracked by a corresponding Jira card in the SRE's backlog so that its implementation can be prioritized and tracked by the SRE leads**
+
+
+Questionnaire to be answered during the PMR:
+* How can we protect ourselves better from such an incident?
+* How can we ensure that the issue does not reoccur?
+* How can we recover quicker from such an incident?
+* How can we identify the issue quicker?
+
+
+Useful resources:
+* A template to be used to document the PMR can be found here.
+* Guidelines on RCA writing can be found here.
+* How to run effective Post Mortem: recording, slides
+* Pager Duty PMR resources
+
+## Detailed process steps
+This section aims at presenting the steps that need to be taken in case of a new incident.
+
+![flow chart](resources/im_flowchart.png)
+
+## Handoff
+Some major incidents will span shift handoff between regions. When this happens:
+
+1. Outgoing Incident Commander identifies an incoming Incident Commander
+2. Incoming Incident Commander identifies an incoming Incident Tech Lead, and any Supporting SRE engineers
+3. Outgoing incident personnel brief by incoming incident personnel at shift handoff. Be sure to include:
+  1. Technical status of the issue
+  2. List of open and eliminated hypothesis regarding root cause
+  3. Time and content of the most recent customer update
+  4. Communication channels in play, including: slack channel, tracking doc, conference bridge, and email thread
+  5. Key support personnel / stakeholders outside SRE-P engaged in the incident, such as CEE TSE, TAM, Account Team, Red Hat executives, or other SRE teams (when troubleshooting hosted tenants or layered products)
```

Comment:
s/SRE-P/SRE
sentence is too Red Hat specific, should be rephrased to be more generic